middleware/validator.js

```diff
@@ -14,6 +14,12 @@ const loginUserSchema = yup.object({
   password: yup.string().required().min(8),
 });
 
+const createProjectSchema = yup.object({
+  name: yup.string().required().max(1024),
+  description: yup.string().required().max(1024),
```

Comment:
Add a minimum condition on the project description as it can be lengthy.
```suggestion
  description: yup.string().required().min(255).optional(),
```